apps/key-manager/src/app/api/key/route.ts

```diff
@@ -0,0 +1,97 @@
+"use server";
+
+import { createHash } from "node:crypto";
+import { auth } from "@/auth";
+import { getCloudflareContext } from "@opennextjs/cloudflare";
+import { createId } from "@paralleldrive/cuid2";
+
+const getUserPrefix = (userId: string) => {
+  const hash = createHash("sha256");
+  const prefix = hash.update(userId).digest("base64url");
+
+  return prefix;
+};
+
+const createUserKeyHelper = async (userId: string) => {
+  const prefix = getUserPrefix(userId);
+  const type = "sk";
+  const uniqueId = createId();
+  const completeKey = `${prefix}:${type}:${uniqueId}`;
+
+  const ctx = await getCloudflareContext();
+  await ctx.env.API_KEYS.put(completeKey, '{"_type":"secret","resources":{"FUZZY_SEARCH":true}}');
```

Comment:
Key identifier generation logic looks mostly correct. I assume this will eventually be worked on but there will be additional logic needed for the key metadata:
- Publishable key vs secret key (and if pk, need set of acceptable origins)
- For admin: granting resources on case-by-case basis, and overriding rate limit